docs/web3-eth-personal.rst

```diff
@@ -258,6 +258,49 @@ Example
     .then(console.log('Account unlocked!'));
     > "Account unlocked!"
 
+------------------------------------------------------------------------------
+
+
+importRawKey
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.personal.importRawKey(privateKey, password)
+
+Imports the given private key into the key store, encrypting it with the passphrase.
+
+Returns the address of the new account.
+
+.. note:: Sending your account password over an unsecured HTTP RPC connection is highly unsecure.
+
+----------
+Parameters
+----------
+
+
+1. ``privateKey`` - ``String`` - An unencrypted private key (hex string).
```

Comment:
Could you rename them to keydata and passphrase just that we have the same naming as in geth. :-)
https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal_importrawkey